src/herder/test/HerderTests.cpp

```diff
@@ -393,10 +393,7 @@ testTxSet(uint32 protocolVersion)
         {
             auto tx = std::static_pointer_cast<TransactionFrame>(
                 txSet->mTransactions[0]);
-            auto& tb = tx->getEnvelope().type() == ENVELOPE_TYPE_TX_V0
-                           ? tx->getEnvelope().v0().tx.timeBounds.activate()
-                           : tx->getEnvelope().v1().tx.timeBounds.activate();
-            tb.maxTime = UINT64_MAX;
+            setMaxTime(tx, UINT64_MAX);
```

Comment:
This PR looks good to me, but I noticed something incorrect in the existing test code. `setMaxTime` and `setMinTime` take an `int64_t` for the time parameter, which is incorrect. Do you mind fixing this? I can also open a separate PR if you'd prefer.

Reply:
Let's do it in a separate PR if you don't mind, because I'm unsure of how wide of a scope that might distort this PR that is otherwise hyperfocused on a single issue.

Reply:
Yeah no problem. I'm opening the PR now.

Reply:
Does that make this PR mergeable?

Reply:
I'd prefer for #3024 to get merged first since this PR passes `UINT64_MAX` to `setMaxTime`.